tests/utils/test_disk_validator.py

```
@@ -32,11 +31,11 @@ def test_validate_disk_space_insufficient(self, tmp_path, monkeypatch):
         f.write_text("data")
         dest = tmp_path / "full_dest"
         dest.mkdir()
-        
+
         # Mock disk usage to return only 1 byte free
         # Required space will be at least 1MB (default buffer)
         monkeypatch.setattr(shutil, "disk_usage", lambda p: (10**9, 10**9 - 1, 1))
```

Ruff (0.15.9)

[warning] Unused lambda argument: `p`

(ARG005)